### PR TITLE
Replace site config key literals with constants

### PR DESCRIPTION
**Pull request by grs**

### CODE REVIEW

File: api/types/types.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ const (
 	DefaultVanName      string = "skupper"
 	DefaultSiteName     string = "skupper-site"
 	ClusterLocalPostfix string = ".svc.cluster.local"
+	SiteConfigMapName   string = "skupper-site"
```
Comment (fgiorgetti) [inline api/types/types.go:30]: I believe the following source files could refer to this new constant:
* client_debug.go:81
* site-controller/controller.go:410
* cli/delete.go:49

File: client/site_config_create.go [hunk 2]
```diff
@@ -12,117 +12,152 @@ import (
...
+	SiteConfigConsoleIngressKey        string = "console-ingress"
+
+	//router options
+	SiteConfigRouterConsoleKey          string = "router-console"
+	SiteConfigRouterLoggingKey          string = "router-logging"
+	SiteConfigRouterDebugModeKey        string = "router-debug-mode"
+	SiteConfigRouterCpuKey              string = "router-cpu"
+	SiteConfigRouterMemoryKey           string = "router-memory"
+	SiteConfigRouterAffinityKey         string = "router-affinity"
+	SiteConfigRouterAntiAffinityKey     string = "router-anti-affinity"
```
Comment (fgiorgetti) [inline client/site_config_create.go:35]: One comment on these `affinity` and `anti-affinity` keys. I noticed that most of the
keys match the command line arguments, but the affinity related ones do not, example:
```
Key = router-affinity
Arg = router-pod-affinity
```

and

```
Key = router-anti-affinity
Arg = router-pod-antiaffinity
```

Not sure if they are intended to match each other. What do you think?

**Review by fgiorgetti — COMMENTED** [review]
@grs A few observations for your evaluation.

File: api/types/types.go (hunk 1 above)
Comment (grs) [reply api/types/types.go:30]: Agreed, fixed in latest update.

File: client/site_config_create.go (hunk 2 above)
Comment (grs) [reply client/site_config_create.go]: Agreed, I changed them in the latest update so they match.

**Review by fgiorgetti — APPROVED** [review]

**Pull request merged**